monitoring-alerting/helm/values.yml

```diff
@@ -0,0 +1,50 @@
+grafana:
+  persistence:
+    enabled: true
+
+  grafana.ini:
+    # users:
+      # viewers_can_edit: true
+    server:
+      root_url: http://grafana.spinnaker.homedepot.com
+    # auth:
+      # disable_login_form: true
+      # disable_signout_menu: true
+    auth.google:
+      enabled: true
+      client_id: CLIENT_ID
```

Comment:
Is the idea here that the `client_id` and `client_secret` in the values yml file be manually updated before running the `helm install`?

Reply:
I went over this in the README: `After running the installation command you'll need to edit the yaml of the `spin-grafana` pod environment variables`. 

The client ID and secret *could* be manually updated beforehand as well. I'd like to figure out a programatic way to do this... but the client ID and secret are generated in google so we'd need to store them in vault or something.

Reply:
Instead of manually editing the spin-grafana pod (which will be overwritten upon the next helm upgrade) consider set the values of `client_id` and `client_secret` in the pipeline or script which will deploy the helm chart.

As helm allows setting of these values, it isn't an ideal practice IMO to edit pods after-the-fact.  I know we don't want to persist the values in the git repo, but we should consider setting those in whatever automation is deploying the helm chart instead of needing to run a `kubectl edit pod ...` command.

Reply:
The `install.sh` script I added populates the client ID and secret in the `values.yaml` file. It first copies the `values.yaml` to `/tmp` to avoid accidentally pushing updated fields to github.